sdk/src/lib.rs

```diff
@@ -34,6 +34,7 @@ pub mod keyed_account;
 pub mod log;
 pub mod native_loader;
 pub mod nonce_account;
+pub mod num_utils;
```

Comment:
I'll echo @ckamm here. Let's move the introduction of this module to the PR where it will be used